docs/development/README.md

```diff
@@ -0,0 +1,97 @@
+# Development
+
+You can find everything related to getting started with development on TFchain here.
+
+## Learn The Basics
```

Comment:
> Learn the basics

As default we should not use capital letters in middle of section name unless it is proper noun or else
I see a lot of cases below

Reply:
fixed